docs/technical/new_instance.md

```diff
@@ -0,0 +1,188 @@
+# Creating a new Insights instance
+The overall idea is strong isolation across districts, which means some initial setup overhead.  Here's how you do it!  More detailed notes for various steps are below.
+
+### 1: Stand up an instance
+Goal: Visit the home page signed-out
+- Add domain
+- SSL certificate
+- New Heroku app with Node/Ruby buildpacks, grant access to team
+- Review and set environment variables (minimally)
+- Review and set PerDistrict values (minimally)
+- Review and add Heroku scheduler jobs (minimally)
+- Set up Postgres and seed database (minimally)
+- Set up Logentries
+- Set up Rollbar
+- Set up Mixpanel
+- Set up CSP and CSP logger
+
+### 2: Sign in
+Goal: Sign in as developer, see empty home page, roster page, districtwide pages
+- Create developer user
+
+### 3: Import first data
+Goal: Sign in as districtwide educator, use the home page, roster page, student profile page, take notes for students in support meetings
+- Create an instance for receiving data (maybe with static IPs, grant access to team)
```

Comment:
You mean "create an SFPT box instance" here, right?

Reply:
This one line is a whole process onto itself: setting up new box, communicating with district parter to connect with SIS, setting up security procedures and updates for the box.

Reply:
Ah yeah I think this file is "new (Insights) instance" and here it's "new (EC2) instance" thanks!

Yep, lots of these things have a bunch of discrete steps, and if there's more details that are helpful to capture we can add those at the bottom of this doc too.  If you have notes on this from when we did, you could push them here or add them in another PR, that'd be awesome.